test/integration/associations/belongs-to.test.js

```diff
@@ -152,6 +152,44 @@ describe(Support.getTestDialectTeaser('BelongsTo'), () => {
         });
       });
     });
+
+    it('supports schemas when defining custom foreign key attribute #9029', function() {
+      const self = this,
```

Comment:
No `self`, with arrow function we dont need special variable to access upper scope